supply-chain/audits.toml

```diff
@@ -65,7 +65,7 @@ notes = "I am the author of this crate."
 
 [[audits.base64]]
 who = "Pat Hickey <phickey@fastly.com>"
-criteria = "safe-to-run"
+criteria = "safe-to-deploy"
```

Comment:
It's my understanding that cargo-vet audits are meant to be "append only", so we should add a new entry for `safe-to-deploy` rather than modifying the existing entry.

Reply:
Ok, I fixed this by making another entry.